src/app/listing/listing.page.html

```diff
@@ -46,7 +46,7 @@ <h6 class="ion-padding-top">{{solution?.description}}</h6>
     </ng-template>
   </ion-grid>
 </ion-content>
-<ion-footer *ngIf="solutionList?.count !== 0">
+<ion-footer *ngIf="!(solutionList?.count === 0 || solutionList?.count === solutionList?.data?.length)">
```

Comment:
if solutionList doesn't have data then can't we directly check the length of data property here as it'll be 0 in that case?

Reply:
then why are we comparing count with the length of the data property?